crates/uv/tests/venv.rs

```diff
@@ -622,29 +523,19 @@ fn verify_nested_pyvenv_cfg() -> Result<()> {
 #[test]
 #[cfg(windows)]
 fn path_with_trailing_space_gives_proper_error() {
-    let context = VenvTestContext::new(&["3.12"]);
+    let context = TestContext::new_with_versions(&["3.12"]);
 
-    let mut filters = context.filters();
-    filters.push((
-        regex::escape(&context.cache_dir.path().display().to_string()).to_string(),
-        r"C:\Path\to\Cache\dir".to_string(),
-    ));
-    // Create a virtual environment at `.venv`.
-    uv_snapshot!(filters, Command::new(get_bin())
-        .arg("venv")
-        .arg(context.venv.as_os_str())
-        .arg("--python")
-        .arg("3.12")
-        .env("UV_CACHE_DIR", format!("{} ", context.cache_dir.path().display()))
-        .env("UV_TEST_PYTHON_PATH", context.python_path.clone())
-        .current_dir(context.temp_dir.path()), @r###"
+    // Set a custom cache directory with a trailing space
+    uv_snapshot!(context.filters(), context.venv()
+        .env("UV_CACHE_DIR", format!("{} ", context.cache_dir.path().display())), @r###"
     success: false
     exit_code: 2
     ----- stdout -----
 
     ----- stderr -----
-    error: failed to open file `C:\Path\to\Cache\dir \CACHEDIR.TAG`
+    error: failed to open file `[CACHE_DIR]/ /CACHEDIR.TAG`
       Caused by: The system cannot find the path specified. (os error 3)
     "###
     );
+    // Note the extra trailing `/` in the snapshot is due to the filters, not the actual output.
```

Comment:
Could you filter it out with an extra `/?` after the path?

Reply:
It's not a part of the match, it's a part of the replacement text.

Reply:
Unless we want to use match groups in the replacement, it's a pain to drop. I'm not sure it's worth it.